src/api/app/components/bs_request_overview_avatars_component.rb

```diff
@@ -25,6 +25,14 @@ def group_avatar_objects
     [@review.group.users, @review.group].flatten
   end
 
+  def package_avatar_objects
+    # if the package explicitly has maintainer or reviewer assigned return them
+    reviewers = MaintainersAndReviewersFinder.new(@review).for_package
+    # otherwise, check the related project for responsible user
+    reviewers = MaintainersAndReviewersFinder.new(@review).for_project if reviewers.present?
```

Comment:
shouldn't we check it `unless reviewers.present?`

Reply:
Ah, that's true. Fixing